sample.env

```diff
@@ -8,6 +8,7 @@ KC_HOST_IP=${DOCKER_HOST_IP}
 
 # Firmware Manager connectivity in the format 'http://endpoint:port'
 FIRMWARE_MANAGER_ENDPOINT=http://${DOCKER_HOST_IP}:8089

```

Comment:
I don't think it is necessary since externally I would view it as the firmware manager still. I also didn't want to change the CV Manager API here. If this is wanted I will change it.